FlowCrypt/Functionality/Services/Compose Message Service/ComposeMessageContext.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ComposeMessageContext.swift
+//  FlowCrypt
+//
+//  Created by Roma Sosnovsky on 17/12/21
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import Foundation
+
+struct ComposeMessageContext: Equatable {
+    var message: String?
+    var recipients: [ComposeMessageRecipient] = []
+    var subject: String?
+    var password: String?
+    var attachments: [MessageAttachment] = []
+}
+
+extension ComposeMessageContext {
+    var hasPassword: Bool {
+        guard let password = password else { return false }
+        return password.isNotEmpty
+    }
```

Comment:
In that case maybe it should also have a `getPassword` that would return a nil on empty string. And make password not externally readable? So that we don't just retrieve `.password` which could be empty string - who knows what kind of vulnerability that may lead to. (encrypting message for a password which is an empty string. OpenPGP.js is probably clever to handle it, but don't want to risk it)

Reply:
I implemented this using custom getter for `messagePassword` - https://github.com/FlowCrypt/flowcrypt-ios/blob/feature/issue-1221-message-password/FlowCrypt/Functionality/Services/Compose%20Message%20Service/ComposeMessageContext.swift#L18